adapters/improvedigital/improvedigital.go

```diff
@@ -36,6 +47,15 @@ func (a *ImprovedigitalAdapter) MakeRequests(request *openrtb2.BidRequest, reqIn
 }
 
 func (a *ImprovedigitalAdapter) makeRequest(request openrtb2.BidRequest, imp openrtb2.Imp) (*adapters.RequestData, error) {
+	// Handle Rewarded Inventory
+	extImp, err := getImpExtWithRewardedInventory(imp)
+	if err != nil {
+		return nil, err
+	}
+	if len(extImp) > 0 {
+		imp.Ext = extImp
```

Comment:
Please set just `is_rewarded_inventory` as this will overwrite entire imp.ext object with any properties that could be there already.

Reply:
`getImpExtWithRewardedInventory` method keep all properties of imp.ext object and only add `is_rewarded_inventory` object when data found at prebid.is_rewarded_inventory.

We also did this way for "Google Additional consent" (https://github.com/ImproveDigital/prebid-server/blob/dealid-rewarded-inventory/adapters/improvedigital/improvedigital.go#L68) based on reviewer feedback.